group_vars/model_ubnt_usw_flex.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+# Factory flash instructions:
+# https://github.com/openwrt/openwrt/commit/a9839697896c4fdf8c44a06bbce466ce52493069
+
+openwrt_version: 22.03-SNAPSHOT
+feed_version: 1.3.0-snapshot
+target: ramips/mt7621
+
+# Our partition is only 7360k, not 8m.
+# We actually need to be below 7200k as the next squashfs padding step is 7400k.
+low_flash: true
+model__packages__to_merge:
+  - "-tcpdump-mini"
+  - "-curl"
+  - "-prometheus-node-exporter-lua"
```

Comment:
Can't you remove something else? Monitoring plugins are essential. :/

Reply:
They depend on `curl` though which is 200 KB (!) :smiling_face_with_tear: I'd rather remove these, than use `low_flash` and lose OPKG, because then I can't install the patched poemgr package.

We also still have Collectd.

Reply:
okay :)